backwardcompatibilityml/widget/app.py

```diff
@@ -0,0 +1,90 @@
+# Copyright (c) Microsoft Corporation
+# Licensed under the MIT License.
+
+import torch
+import torch.optim as optim
+import torch.nn.functional as F
+import random
+from backwardcompatibilityml import loss as bcloss
+from backwardcompatibilityml.helpers import training
+from backwardcompatibilityml.helpers.models import LogisticRegression, MLPClassifier
+from rai_core_flask.flask_helper import FlaskHelper
+from .compatibility_analysis import CompatibilityAnalysis
```

Comment:
Can you change this to an absolute import rather than relative.

Reply:
And can we move this file to a different folder, so that it is outside of the widget? Basically so that it doesn't get packaged with the widget.

I'd suggest something like:
```
root_folder
|
|--development
     |
     |--app.py
     |--development-artifacts
     |--public
```